lib/Controller/Report.php

```diff
@@ -104,7 +110,7 @@ class Report extends Base
      * @param LayoutFactory $layoutFactory
      * @param UserFactory $userFactory
      */
-    public function __construct($log, $sanitizerService, $state, $user, $help, $date, $config, $store, $timeSeriesStore, $reportService, $reportScheduleFactory, $savedReportFactory, $mediaFactory, $layoutFactory, $userFactory)
+    public function __construct($log, $sanitizerService, $state, $user, $help, $date, $config, $store, $timeSeriesStore, $reportService, $reportScheduleFactory, $savedReportFactory, $mediaFactory, $layoutFactory, $userFactory, $campaignFactory)
```

Comment:
I think our recent refactor means we don't need $mediaFactory, $layoutFactory or $campaignFactory in this controller?